CHANGELOG.md

```diff
@@ -6,7 +6,10 @@
 
 ##### Enhancements
 
-* None.
+* `VariableNameRule` now accepts symbols starting with more than one uppercase
+  letter to allow for names like XMLString or MIMEType.
```

Comment:
missing two trailing spaces, according to the guidelines at https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes